<ui version="4.0" >
 <class>filtersConfig</class>
 <widget class="QWidget" name="filtersConfig" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>431</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="6" column="0" >
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="1" >
    <widget class="KEditListWidget" name="filtersListWidget" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2" >
    <widget class="QLabel" name="label_2" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Filters settings:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1" >
    <widget class="QRadioButton" name="useRegularExpressionsRadioButton" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>Use regular expressions</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1" >
    <widget class="QRadioButton" name="useExactMatchRadioButton" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>Use exact match</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="12" column="1" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>25</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="2" >
    <widget class="QLabel" name="label" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Filters:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QCheckBox" name="showOnlyMatchesCheckBox" >
     <property name="text" >
      <string>Show only lines that match filters</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <spacer name="verticalSpacer_2" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KEditListWidget</class>
   <extends>QWidget</extends>
   <header>keditlistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>showOnlyMatchesCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>61</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel" >
     <x>10</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showOnlyMatchesCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>filtersListWidget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>170</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel" >
     <x>161</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showOnlyMatchesCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>69</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel" >
     <x>12</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showOnlyMatchesCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>useRegularExpressionsRadioButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>76</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel" >
     <x>177</x>
     <y>368</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showOnlyMatchesCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>useExactMatchRadioButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>76</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel" >
     <x>117</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
